config/channels/in-memory-channel/500-controller.yaml

```diff
@@ -11,11 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: imc-controller
   namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
```

Comment:
Have you nabbed our release script to replace these with a version tag?

Reply:
Not yet. Prepping to do so.